core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -114,4 +114,24 @@ package object config {
   private[spark] val PYSPARK_PYTHON = ConfigBuilder("spark.pyspark.python")
     .stringConf
     .createOptional
+
+  private[spark] val SPARK_IO_ENCRYPTION_ENABLED = ConfigBuilder("spark.io.encryption.enabled")
```

Comment:
nit: can you remove `SPARK_` from the name of these constants? It's kinda obvious that they're part of Spark.